crates/vault-registry/rpc/src/lib.rs

```diff
@@ -16,15 +16,20 @@ use sp_runtime::{
 use std::sync::Arc;
 
 #[rpc]
-pub trait VaultRegistryApi<BlockHash, VaultId, Balance, UnsignedFixedPoint, CurrencyId>
+pub trait VaultRegistryApi<BlockHash, VaultId, Balance, UnsignedFixedPoint, CurrencyId, AccountId, Vault>
```

Comment:
To be *super* pedantic:
```suggestion
pub trait VaultRegistryApi<AccountId, Balance, BlockHash, CurrencyId, VaultId, Vault, UnsignedFixedPoint>
```

Reply:
Haha wow, that _is_ super pedantic. We didn't sort them in the other apis, so I don't think it's worth spending the time to sort them here